### PR TITLE
Feat/383555  clear soft deleted records

### PR DESCRIPTION
**Pull request by whitewaterdesign**

### CODE REVIEW

File: app/config/index.js [hunk 1]
```diff
@@ -25,13 +26,13 @@ const config = {
   robotSheetName: process.env.ROBOT_SHEET_NAME,
   osPlacesApi: {
     baseUrl: process.env.OS_PLACES_API_BASE_URL,
-    token: process.env.OS_PLACES_API_KEY
+    token: getEnvironmentVariable('OS_PLACES_API_KEY')
```
Comment (jbarnsley10) [inline app/config/index.js:29]: If you're changing the calls to use your new function, should you not make all values use this function for consistency?

Comment (whitewaterdesign) [reply app/config/index.js:29]: we can do, I've been doing it on an as needed basis at the moment.  It will make testing more difficult otherwise.

**Review by jbarnsley10 — CHANGES_REQUESTED** [review]
Just the consistency of getEnvironmentVariable being used for all values

**Review by jbarnsley10 — APPROVED** [review]
LGTM

**Pull request merged**